extensions/create-link/src/copy-html.ts

```diff
@@ -0,0 +1,40 @@
+import { showHUD, Clipboard } from "@raycast/api";
+import { isExtensionInstalled, getActiveTab } from "./utils/browser";
+
+function sanitizeForHtml(text: string): string {
+  return text
+    .replace(/&/g, "&amp;")
+    .replace(/</g, "&lt;")
+    .replace(/>/g, "&gt;")
+    .replace(/"/g, "&quot;")
+    .replace(/'/g, "&#039;");
+}
+
+function sanitizeUrl(url: string): string {
+  try {
+    new URL(url);
+    return url;
+  } catch {
+    return "about:blank";
+  }
+}
+
+export default async function copyHTML() {
+  if (!isExtensionInstalled()) {
+    await showHUD("Extension not installed");
+    return;
+  }
+
+  const activeTab = await getActiveTab();
```

Comment:
![](https://github.com/user-attachments/assets/c99cdcda-1292-40a1-8a86-a5dca5e7649e)

You've thrown a new error in the function.